src/test/lrsql/admin/route_test.clj

```diff
@@ -291,6 +298,30 @@
     (testing "omitted sec headers because not configured"
       (let [{:keys [headers]} (get-env content-type)]
         (is (empty? (select-keys headers sec-header-names)))))
+    (testing "delete actor route"
+      (let [stmt-0 {"id"      "00000000-0000-4000-8000-000000000000"
+                    "actor"   {"mbox"       "mailto:sample.foo@example.com"
+                               "objectType" "Agent"}
+                    "verb"    {"id"      "http://adlnet.gov/expapi/verbs/answered"
+                               "display" {"en-US" "answered"
+                                          "zh-CN" "回答了"}}
+                    "object"  {"id" "http://www.example.com/tincan/activities/multipart"}
+                    "context" {"platform" "example"}}
+            lrs (:lrs sys')
+            auth-ident {:agent  {"objectType" "Agent"
+                                 "account"    {"homePage" "http://example.org"
+                                               "name"     "12341234-0000-4000-1234-123412341234"}}
```

Comment:
Same story with this agent (I think we may already have `def`s of agents, though I don't recall off the top of my head)

Reply:
done